packages/components/src/components/option/option.stories.ts

```diff
@@ -34,149 +35,147 @@ export const Default = {
+ * Use the `variant` attribute to set the option’s variant.
+ * - `default`
+ * - `checkbox`: Enabled automatically when using `sd-select` with attribute `checklist` set to `true`.
+ *
+ * **Hint:**
+ * Option can only be selected when utilized in sd-select.
  */
 
-export const DisabledCheckbox = {
-  name: 'Disabled x Checkbox',
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'disabled',
-          values: [false, true]
-        },
-        y: {
-          type: 'attribute',
-          name: 'checkbox',
-          values: [false, true]
-        }
-      },
-      constants: { type: 'attribute', name: 'disabled', value: true },
-      args
-    });
-  }
+export const Variants = {
+  render: () =>
+    html`<div class="flex flex-row gap-12">
+      <sd-select
+        class="h-[260px] w-[420px]"
+        label="Default"
+        placeholder="Please Select"
+        placement="bottom"
+        max-options-visible="3"
+      >
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+      </sd-select>
+      <sd-select
+        class="h-[160px] w-[420px]"
+        label="Checkbox"
+        placeholder="Please Select"
+        placement="bottom"
+        max-options-visible="3"
+        multiple
+      >
+        <sd-option value="option-1" checkbox>Checkbox Option 1</sd-option>
+        <sd-option value="option-2" checkbox>Checkbox Option 2</sd-option>
+        <sd-option value="option-3" checkbox>Checkbox Option 3</sd-option>
+      </sd-select>
+    </div>`
 };
 
 /**
- * Use the `size` attribute to change a select’s size. This will be inherited automatically from the `size` attribute of the parent `sd-select`.
+ * Use the `size` attribute to change the option’s size.
+ * This will be inherited automatically from the size attribute of the parent `sd-select`.
  */
 
-export const SizeCheckbox = {
-  name: 'Size x Checkbox',
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'size'
-        },
-        y: {
-          type: 'attribute',
-          name: 'checkbox',
-          values: [false, true]
-        }
-      },
-      constants: { type: 'attribute', name: 'checkbox', value: true },
-      args
-    });
-  }
+export const Size = {
+  render: () =>
+    html` <div class="flex flex-row justify-between">
+      <sd-select
+        class="h-[260px] w-[220px]"
+        label="Large"
+        placeholder="Please Select"
+        placement="bottom"
+        max-options-visible="3"
+      >
+        <sd-option value="option-1" size="lg">Large Option 1</sd-option>
+        <sd-option value="option-2" size="lg">Large Option 2</sd-option>
+        <sd-option value="option-3" size="lg">Large Option 3</sd-option>
+      </sd-select>
+      <sd-select
```

Comment:
sd-select itself also needs to be size medium

Reply:
![image](https://github.com/user-attachments/assets/f29ca0b8-e768-41eb-a857-41ad23dde62d)